packages/core/src/messagingProtocols/index.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Wire
+ * Copyright (C) 2022 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+export * from './mls';
+export * from './proteus';
```

Comment:
I find it a little confusing that we mix the two protocols together. 
We get mls and proteus function all together and I find it hard to understand which function is mls or proteus specific. 

I think they should both live in a different namespace (aka removing this `index.ts` and import file using `messagingProtocol/mls` or `messagingProtocol/proteus`)